src/pe_reports/report_generator.py

```diff
@@ -264,13 +259,15 @@ def main():
     if not os.path.exists(validated_args["OUTPUT_DIRECTORY"]):
         os.mkdir(validated_args["OUTPUT_DIRECTORY"])
 
+    try:
+        soc_med = validated_args["--soc_med_included"]
+    except Exception as e:
+        LOGGER.info(f"Social media should not included: {e}")
+        soc_med = False
     # Generate reports
-    generated_reports = generate_reports(
-        validated_args["REPORT_DATE"],
-        validated_args["OUTPUT_DIRECTORY"],
+    generate_reports(
+        validated_args["REPORT_DATE"], validated_args["OUTPUT_DIRECTORY"], soc_med
```

Comment:
Related to suggestion above.  Note, I didn't run this thru `black`, so it might get reformatted due to long line length:
```suggestion
        validated_args["REPORT_DATE"], validated_args["OUTPUT_DIRECTORY"], validated_args["--soc_med_included"]
```